concordium-std/src/impls.rs

```diff
@@ -678,3 +685,111 @@ impl<A: fmt::Debug> ExpectNoneReport for Option<A> {
         }
     }
 }
+
+impl<K: Serial + Ord> SerialCtx for BTreeSet<K> {
+    fn serial_ctx<W: Write>(
+        &self,
+        size_len: &schema::SizeLength,
```

Comment:
Since `SizeLength` is a small type it should be taken by value, not by reference.